client/src/components/layout/IndexPage.js

```diff
@@ -165,8 +171,9 @@ const IndexPage = (props) => {
         </div>
       </form>
       {table}
+      {user ? authenticatedUserSaveTable : unauthenticatedUser}
```

Comment:
I'm a bit confused by the logic here - are only authenticated users able to persist and save a table but a visitor can see a table, just not save it?

Reply:
Yes, exactly